src/docs/development/platform-integration/web.md

```diff
@@ -9,32 +9,36 @@ As of the 1.12 release, web support for Flutter
 is available on the beta channel.
 **We don't recommend deploying a web app to production.**
```

Comment:
Can we update this question while we're tackling this page? We've made a lot of progress since 1.12, and we have real customers in production (including our own product), so we should start to turn the dial a bit. 

I'd say: 
> Flutter web support is now available in the `beta` channel. As is typical for beta software, we're still making changes to improve performance and quality, and you may discover rendering bugs compared to the stable mobile platforms. While the web support is in beta, we recommend careful testing on a variety of platforms before deploying to production, and that you track and apply Flutter updates to your deployed site on a regular basis.